Library/src/Systems/Assets/AssetSystem.cpp

```diff
@@ -847,4 +851,36 @@ CSP_ASYNC_RESULT_WITH_PROGRESS void
 	GetAssetsByCriteria({InAsset.AssetCollectionId}, nullptr, nullptr, Array<EAssetType> {EAssetType::MODEL}, GetAssetsCallback);
 }
 
+CSP_EVENT void AssetSystem::SetAssetDetailBlobChangedCallback(AssetDetailBlobChangedCallbackHandler Callback)
+{
+	AssetDetailBlobChangedCallback = Callback;
+	RegisterSystemCallback();
+}
+
+void AssetSystem::RegisterSystemCallback()
+{
+	if (!AssetDetailBlobChangedCallback)
+	{
+		return;
+	}
+
+	EventBusPtr->ListenEvent("AssetDetailBlobChanged", this);
+}
+
+void AssetSystem::DeregisterSystemCallback()
+{
+	if (EventBusPtr)
```

Comment:
Add parenthesis here, and in the method below - it's one of our conventions.

Reply:
done